### ISSUE TITLE
Reuse training set kernel for testing set

### ISSUE THREAD

**Issue by mirix** [AnotherSamWilson/miceforest#45]
Hello,

I have not been able to find much documentation on the impute_new_data function.

Can one save a kernel as, say, pickle to then load it in a different python script?

If possible, any hints on how to proceed?

In your opinion, would it make sense (pros and cons) to reuse the kernel from the training set in order to impute the testing set?

**Comment by AnotherSamWilson** [comment]
There is a save_kernel method in version 5.4.0 and later, and a load_kernel function in miceforest.utils. I recommend using this, it will compress the kernel quite a bit.

Conventional wisdom is to use the training kernel to impute the test set yes, however it really depends on your use case. If you want the most accurate imputations you can get, imputing the data all at once will probably yield better results.
